string operation
float32 num1
float32 num2
---
bool onList
float32 answer
